doc/rdoc/markup_reference.rb

```diff
@@ -29,16 +29,31 @@
 #   see <tt>:nodoc:</tt>, <tt>:doc:</tt>, and <tt>:notnew</tt>.
 # - \RDoc directives in single-line comments;
 #   see other {Directives}[rdoc-ref:RDoc::MarkupReference@Directives].
-# - The Ruby code itself;
+# - The Ruby code itself (but not from C code);
 #   see {Documentation Derived from Ruby Code}[rdoc-ref:RDoc::MarkupReference@Documentation+Derived+from+Ruby+Code]
```

Comment:
I realize this isn't in your change, but this line is missing a period at the end.

Reply:
Fixed.